.env.example

```
@@ -1,2 +1,4 @@
 DISCORD_TOKEN=
-GUILD_ID=
\ No newline at end of file
+GUILD_ID=
+PARTNER_ID=
+PARTNER_KEY=
```

dotenv-linter (3.3.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)